src/gwt/src/org/rstudio/studio/client/workbench/views/packages/Packages.java

```diff
@@ -1269,7 +1058,6 @@ public int compare(PackageInfo o1, PackageInfo o2)
       packratContext_ = newState.getPackratContext();
       view_.setProgress(false);
       setViewPackageList();
-      setViewActions(newState);
```

Comment:
It's expensive to and complicated to compute the set of actions, so if we aren't going to expose them, we should also excise the code which generates them. I think the gentlest change here (from the user's perspective) would be to slightly reorganize the code so that the packrat actions, instead of being eagerly fetched with the package state, is fetched via a separate RPC that the user manually invokes by clicking a Packrat sync button. In that world we'd still have all of this code (generating actions and presenting UX for them), it just wouldn't light up unless you asked for it.